main.tf

```diff
@@ -0,0 +1,193 @@
+# TODO: (input) Create ACM cert outside of this module
+
+# TODO: use `terraform-terraform-label` to generate all labels.
+
+# TODO: (output) security group IDs
+
+#module "container_definition" {
+#  source    = "git::https://github.com/cloudposse/terraform-aws-ecs-container-definition.git?ref=init"
+#  container_name      = "app"
+#  container_image    = "nginx"
+#}
+
+# ECR repository
+resource "aws_ecr_repository" "app" {
+  name = "${var.app_name}"
+}
+
+# Cloudwatch Log Group
+resource "aws_cloudwatch_log_group" "app" {
+  name = "${var.app_name}"
+
+  tags {
+    Stage = "${var.stage}"
+    Application = "${var.app_name}"
+  }
+}
+
+# ECS Task def
+data "template_file" "web_task" {
+  template = "${file("task_definition.json")}"
+
+  vars {
+    image     = "${aws_ecr_repository.app.repository_url}"
+    log_group = "${aws_cloudwatch_log_group.app.name}"
+  }
+}
+
+resource "aws_ecs_task_definition" "web" {
+  family                = "${var.stage}_web"
```

Comment:
hrm, I think we'll leave `stage` out. Move `family` to var and just default it to `web`